nservicebus/sqlserver/addressing_content_sqlserver_[,2.1).partial.md

```diff
@@ -0,0 +1,19 @@
+### Format
+
+The SQL Server address has following canonical form
+
+```
+table
+```
+
+The table name is an bracket-delimited identifier without the surrounding square brackets. Whitespace characters are allowed. Brackets must be escaped e.g. `my table` and `my]]table` are legal values. The surrounding brackets are added automatically by SQL Server transport when executing the T-SQL statements. `@` is not a valid character. All characters after `@` (including `@`) are omitted when parsing addresses e.g. `my t@ble` is parsed as `my t`. 
```

Comment:
It is only duplicated in two versions so I don't think it is worth the effort of maintaining a separate file.